mne/viz/topomap.py

```diff
@@ -703,6 +703,57 @@ def _inside_contour(pos, contour):
     return check_mask
 
 
+def _plot_ica_topomap(ica, idx=0, ch_type=None, res=64,
+                      layout=None, vmin=None, vmax=None, cmap='RdBu_r',
+                      sensors=True, colorbar=False, title=None,
+                      show=True, outlines='head', contours=6,
+                      image_interp='bilinear', head_pos=None, axis=None):
+    """plot single ica map to axes"""
+    import matplotlib as mpl
+    from ..channels import _get_ch_type
+
+    if ica.info is None:
+        raise RuntimeError('The ICA\'s measurement info is missing. Please '
+                           'fit the ICA or add the corresponding info object.')
+    if not isinstance(axis, mpl.axes.Axes):
+        raise ValueError('axis has to be an instance of matplotlib Axes, '
+                         'got %s instead.' % type(axis))
+    ch_type = _get_ch_type(ica, ch_type)
+
+    data = np.dot(ica.mixing_matrix_[:, [idx]].T,
+                  ica.pca_components_[:ica.n_components_])
```

Comment:
currently it is a dupe, but I was planning to refactor other functions to use this part (it allows to plot to specified axis) - I just left it for another PR. See the discussion. I can refactor now of course.